pom.xml

```diff
@@ -95,12 +95,15 @@
         specified. Used by the `patch` profile to enable patching. -->
         <error-prone.args />
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <version.auto-service>1.0-rc7</version.auto-service>
         <version.error-prone>2.4.0</version.error-prone>
         <version.error-prone-javac>9+181-r4173-1</version.error-prone-javac>
         <version.guava-beta-checker>1.0</version.guava-beta-checker>
+        <version.immutables>2.8.8</version.immutables>
         <version.jdk>1.8</version.jdk>
         <version.jooq>3.13.2</version.jooq>
         <version.maven>3.6.1</version.maven>
```

Comment:
Realized after approving it, should we upgrade the Maven version before the release?

Reply:
We can, sure. Separate PR? :)

Reply:
Yes, let's do it in a separate PR.